renderer/RendererLib/RendererAPI/include/RendererAPI/Types.h

```diff
@@ -74,8 +74,8 @@ namespace ramses_internal
     struct WindowsWindowHandleTag {};
     using WindowsWindowHandle = StronglyTypedValue<void *, nullptr, WindowsWindowHandleTag>;
 
-    struct AndroidNativeWindowPtrTag {};
-    using AndroidNativeWindowPtr = StronglyTypedValue<void *, nullptr, AndroidNativeWindowPtrTag>;
+    struct MobilePlatformNativeWindowPtrTag {};
```

Comment:
this is a breaking change, unfortunately... Should add a new type and new API to set the iOS/apple native handle

Reply:
Ok, I reverted the change and added `IOSNativeWindowPtr` as a new field everywhere where `AndroidNativeViewPtr` existed.